gpt4all-api/gpt4all_api/app/api_v1/routes/chat.py

```diff
@@ -1,26 +1,45 @@
 import logging
 import time
-from typing import List
+from typing import List, Optional
```

Comment:
It'd make more sense to follow [PEP 604](https://peps.python.org/pep-0604/) and use `Type | None` instead of `Optional[Type]`. You'll need to `from __future__ import annotations` for that.

Reply:
Thanks for the heads up, I will work on this.